packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/direct-edit/DiagramDirectEditContext.tsx

```diff
@@ -28,7 +35,7 @@ export const DiagramDirectEditContextProvider = ({ children }: DiagramDirectEdit
   });
 
   const setCurrentlyEditedLabelId = useCallback(
-    (directEditTrigger: DirectEditTrigger, currentlyEditedLabelId: string, editingKey: string | null) => {
+    (directEditTrigger: DirectEditTrigger | null, currentlyEditedLabelId: string | null, editingKey: string | null) => {
```

Comment:
Do we really have a use case with null for those values? Apart from this new default value? I don't see why we should change what we expect

Reply:
We kind of need it to reset default value on direct edit close.
I have added a reset direct edit function to the context